### DISCUSSION TITLE
Maui alive or...

### DISCUSSION THREAD

**Discussion by Xyncgas** [dotnet/maui#14727] [Ideas]
Is it still being worked on? How many people are maintaining this project from microsoft? Should we move to flutter instead

How are you going to explain it's production ready, but have bugs that are being looked at very slowly, for example the bug in WindowsSDK that was fixed that was causing Mouse to be unusable https://github.com/MicrosoftEdge/WebView2Feedback/issues/3003 requires people to manually change their dependency in the project file, perhaps you can bump it instead of adding it to backlog since it's more like a P1 issue... (https://github.com/dotnet/maui/issues/14107)

What are you saying to people who trusted you, are you confident the project is at a healthy state, are you confident it's getting there, is now the moment we should dip, is there any reason to stick around, is microsoft taking responsibility for this project or not

**Comment by KarthikHebbarMN** [comment]
I would say move to flutter. We might just as well do that.

**Comment by Xyncgas** [comment]
Everyone try out F# on Fable it's amazing

**Reply by LuohuaRain** [reply]
Really? This is the first time I've heard of it. Can it compete with MAUI or Flutter?

**Reply by Xyncgas** [reply]
Their tutorial includes using F# to write the pac man game, on the browser side it's a canvas, the mouse input and keyboard input are intercepted and you can click on the game's menu

I recommend using the solution over MAUI because you don't need to deal with Xamari-style if that's not your thing and F# writes like python and I say it can do what flutter can do
